pykickstart/commands/module.py

```diff
@@ -188,3 +188,7 @@ def _getParser(self):
 
         """ % versionToLongString(RHEL10))
         return op
+
+class F41_Module(RHEL10_Module):
```

Comment:
This can't inherit from RHEL10, if it does then the deprecation message is going to be wrong.
RHEL messages can reference Fedora since they are based on them, but Fedora cannot reference RHEL, it needs to say it was deprecated in Fedora 41.